### PR TITLE
Update tasmanian

### PR DESCRIPTION
**Pull request by aerijman**
FOR CONTRIBUTOR:
* [x ] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x ] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/presto/presto_macros.xml [hunk 1]
```diff
@@ -42,7 +42,6 @@
     
     <token name="@HELP_NOTE@"><![CDATA[
 
-====
```
Comment (bgruening) [inline tools/presto/presto_macros.xml:45]: please do not change other tools in this PR

File: tools/tasmanian_mismatch/Tasmanian.xml [hunk 2]
```diff
@@ -1,6 +1,6 @@
-<tool id="tasmanian_mismatch" name="Analysis of artifacts with Tasmanian" version="0.1.3" profile="18.09">
+<tool id="tasmanian_mismatch" name="Analysis of artifacts with Tasmanian" version="1.0.7" profile="18.09">
   <requirements>
-    <requirement type="package" version="0.1.3">tasmanian-mismatch</requirement> 
+    <requirement type="package" version="1.0.7">tasmanian-mismatch</requirement>
```
Comment (bgruening) [inline tools/tasmanian_mismatch/Tasmanian.xml:3]: If you want to use version macros as described here: https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html?highlight=version#tool-versions

Example here: https://github.com/galaxyproject/tools-iuc/blob/master/tools/seqwish/seqwish.xml#L3

We have a bot that helps you updating the Galaxy tools as well.

File: tools/presto/presto_macros.xml (hunk 1 above)
Comment (aerijman) [reply tools/presto/presto_macros.xml]: Oops and thank you. Fixed

File: tools/tasmanian_mismatch/Tasmanian.xml (hunk 2 above)
Comment (aerijman) [reply tools/tasmanian_mismatch/Tasmanian.xml]: Did it as in the example. Thanks!

Comment (aerijman) [reply tools/tasmanian_mismatch/Tasmanian.xml]: Is there a way to point to the latest version in conda without the need to hard code the version?

**Comment by aerijman** [comment]
Is there a linting problem (which I didn't experience locally) or divergent histories? How should I solve it?

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks @aerijman!

**Comment by aerijman** [comment]
@bgruening Thank you very much!

**Pull request merged**